lib/src/constants/assets.dart

```diff
@@ -7,21 +7,21 @@ class Assets {
 
   // Visa types logo
   /// Logo for Visa
-  static const visaLogo = '$imagePath/visa_logo.png';
+  static String visaLogo = '$imagePath/visa_logo.png';
```

Comment:
Reason for it?

`const` ensures the value is immutable at compile-time, which can help avoid accidental modifications. Changing to `String` allows the possibility of `visaLogo` being modified, which could introduce bugs if unintended.